tsconfig.json

```diff
@@ -5,17 +5,18 @@
     "module": "commonjs",
     "moduleResolution": "node",
     "lib": ["es6", "es7"],
+    "skipLibCheck": true,
```

Comment:
Is this really needed?

Reply:
All changes in `tsconfig.json` are temporary until we implement langserver properly. Though we do need to have either `declaration` or `allowJs` enabled because they are mutually exclusive.

Reply:
I don't think we need to specify either of those.